terraform/helm/aptos-node/templates/validator.yaml

```diff
@@ -208,4 +212,8 @@ spec:
   - ports:
     - protocol: TCP
       port: 443
+  # Logging
+  - ports:
+    - protocol: TCP
+      port: 5044
```

Comment:
this assumes struct log shipper is at 5044. maybe use helm values to be able to configure struct log shipper address and port?

Reply:
we didn't make the port configurable on the logger helm chart yet. would be a good followup

Reply:
maybe even make a good first issue for this